rocketmq-spring-boot/src/test/java/org/apache/rocketmq/spring/autoconfigure/RocketMQAutoConfigurationTest.java

```diff
@@ -272,5 +293,24 @@ public RocketMQTemplate extRocketMQTemplate() {
     static class MyExtRocketMQTemplate extends RocketMQTemplate {
```

Comment:
Could you modify the class MyExtRocketMQTemplate name, you could assign a concrete name Instead of a name with no actual meaning my**?

Reply:
done : )